polylint.js

```diff
@@ -28,16 +28,20 @@ var polylint = function polylint(path, options) {
   options.filter = function(){
     return false;
   };
-  options.redirect = "bower_components";
+  if (!('redirect' in options)) {
+    options.redirect = "bower_components";
+  }
   return hydrolysis.Analyzer.analyze(path, options).then(function(analyzer){
-    var lintErrors = [];
-    for (var linter in linters) {
-      lintErrors = lintErrors.concat(linters[linter](analyzer, path));
+    var allWarnings = [];
+    for (var linterName in linters) {
+      var linter = linters[linterName];
+      var warnings = linter(analyzer, path, options);
```

Comment:
Here I add options as a third argument.  I checked lib/linters.js and all the existing linter functions have arity 1 or 2.